cli/src/plugins/javadoc/index.ts

```diff
@@ -182,6 +183,57 @@ function makeSuperclassList(project: Project, doc: ParsedClassDoc) {
   ];
 }
 
+function makeInterfaceMethodList(project: Project, doc: ParsedClassDoc, interfaceTypes: Type[], inheritedMethods: {[k: string]: string[]}) {
+  if (interfaceTypes.length === 0) {
+    return [];
+  }
+  return [
+    md.heading(2, md.text("Methods Inherited from Interfaces")),
+    md.root(interfaceTypes.map((interfaceType) => {
+      return [
+        md.heading(3, md.text(interfaceType.qualifiedTypeName)),
+        md.list(
+          "unordered",
+          inheritedMethods[interfaceType.qualifiedTypeName].map((method) => {
+            return md.listItem(project.linkToEntity([interfaceType.qualifiedTypeName, method].join('.'), method))
```

Comment:
Unfortunately the method name won't include the simple parameter list so these links won't work?